src/main/java/com/wynntils/hades/protocol/packets/server/HSPacketUpdateMutual.java

```diff
@@ -10,29 +10,42 @@
  * Used for updating mutual friends/party users their status.
  */
 public class HSPacketUpdateMutual implements HadesPacket<IHadesClientAdapter> {
-
     UUID user;
+    String name;
     double x, y, z;
     double health, mana;
     double maxHealth, maxMana;
+    String world;
+    boolean isMutualFriend;
+    boolean isMutualPartyMember;
+    boolean isMutualGuildMember;
```

Comment:
hmmm, I feel like these 3 states are completely unnecessary, let the server decide which state it should be displayed by priority and use an enum instead so we also save some egress bandwidth.